.gitignore

```diff
@@ -40,3 +40,4 @@ packages/babel-preset-env-standalone/babel-preset-env.min.js
 /codemods/*/node_modules
 /packages/babylon/build
 .idea/
+.vscode/
```

Comment:
Can you remove this?

Reply:
We actually have sublime-text and intellij, I don't mind adding vscode as well.

Reply:
I know it's somewhat subjective, but I personally prefer not having any editor/OS specific stuff in the gitignore and add instructions for people to ignore them on their local machine, globally. Not really a big issue worth bikeshedding tho :)

Reply:
I agree @existentialism, I would say either we accept this one or remove all of them (in another PR), what do you think?